### PR TITLE
feat: added user avatars in author search dropdown

### LINKED ISSUE #4193: Add user avatars in author search dropdown
When users have similar names the avatars are useful to quickly distinguish, we use them everywhere else so they should be here too.

### PR DESCRIPTION
**Pull request by adityash1**
Fixes #4193

### CODE REVIEW

File: app/scenes/Search/components/UserFilter.tsx [hunk 1]
```diff
@@ -22,13 +23,17 @@ function UserFilter(props: Props) {
 
   const options = React.useMemo(() => {
     const userOptions = users.all.map((user) => ({
+      user,
       key: user.id,
       label: user.name,
+      icon: <Avatar model={user} />,
```
Comment (adityash1) [inline app/scenes/Search/components/UserFilter.tsx:29]: @tommoor right now it's showing default ring on author. should we change it to something else?

File: app/scenes/Search/components/UserFilter.tsx [hunk 2]
```diff
@@ -24,11 +25,13 @@ function UserFilter(props: Props) {
     const userOptions = users.all.map((user) => ({
       key: user.id,
       label: user.name,
+      icon: <Avatar model={user} />,
     }));
     return [
       {
         key: "",
         label: t("Any author"),
+        icon: <Avatar />,
```
Comment (adityash1) [inline app/scenes/Search/components/UserFilter.tsx:34]: @tommoor right now it's showing a hollow ring. Should we change it to something else?

Comment (tommoor) [reply app/scenes/Search/components/UserFilter.tsx:34]: How about `<UserIcon />` ?

**Comment by tommoor** [comment]
I think you got it this time 🙂

**Pull request closed without merging**